src/android/java/io/jxcore/node/ConnectivityMonitor.java

```diff
@@ -8,43 +8,52 @@
 import android.net.ConnectivityManager;
```

Comment:
I can't find any references anywhere in the code that call this class. Unless we are calling this by reflection it is not being used. In that case can we remove it and its reference in plugin.xml?